docs/source/ecf.rst

```diff
@@ -945,15 +945,31 @@ The following three parameters control the use of pre-generated limb darkening c
 
 use_generate_ld
 ^^^^^^^^^^^^^^^
-If you want to use the generated limb-darkening coefficients from Stage 4, use exotic-ld or spam. Otherwise, use None. Important: limb-darkening coefficients are not automatically fixed, change the limb darkening parameters to 'fixed' in the .epf file if they should be fixed instead of fitted! The limb-darkening laws available to exotic-ld and spam are linear, quadratic, 3-parameter and 4-parameter non-linear.
+If you want to use the generated limb-darkening coefficients from Stage 4, use ``exotic-ld`` or ``spam``. Otherwise, use ``None``. Important: limb-darkening coefficients are not automatically fixed, change the limb darkening parameters to ``'fixed'`` in the .epf file if they should be fixed instead of fitted! The limb-darkening laws available to exotic-ld and spam are linear, quadratic, 3-parameter, and 4-parameter (non-linear).
 
 ld_file
 ^^^^^^^
-If you want to use custom calculated limb-darkening coefficients, set to the fully qualified path to a file containing limb darkening coefficients that you want to use. Otherwise, set to None. Note: this option only works if use_generate_ld=None. The file should be a plain .txt file with one column for each limb darkening coefficient and one row for each wavelength range.
+If you want to use custom calculated limb-darkening coefficients, set to the fully qualified path to a file containing limb-darkening coefficients that you want to use. Otherwise, set to ``None``. Note: this option only works if ``use_generate_ld=None``. The file should be a plain .txt file with one column for each limb-darkening coefficient and one row for each wavelength range. Important: limb-darkening coefficients are not automatically fixed, change the limb darkening parameters to ``'fixed'`` in the .epf file if they should be fixed instead of fitted!
 
 ld_file_white
 ^^^^^^^^^^^^^
-The same type of parameter as ld_file, but for the limb-darkening coefficients to be used for the white-light fit. This parameter is required if ld_file is not None and any of your EPF parameters are set to white_free or white_fixed. If no parameter is set to white_free or white_fixed, then this parameter is ignored.
+The same type of parameter as ``ld_file``, but for the limb-darkening coefficients to be used for the white-light fit. This parameter is required if ``ld_file`` is not None and any of your EPF parameters are set to ``white_free`` or ``white_fixed``. If no parameter is set to ``white_free`` or ``white_fixed``, then this parameter is ignored.
+
+recenter_ld_prior
```

Comment:
Is there any reason we can't re-center priors when using the Exotic-ld coefficients?

Reply:
That's exactly what this parameter was made to do. If you're fixing your LD parameters to any LD model, then you don't need this parameter. But if you want to use a Gaussian prior around any LD model, then you need to set this parameter to True

Reply:
Oh I see what you're saying now - let me fix that

Reply:
Okay, should be good now. Sorry about the confusion - reading/writing this from my phone on the airplane